src/index.js

```diff
@@ -69,7 +71,9 @@ module.exports = class BabiliPlugin {
 
               // do the transformation
               const result = _babel.transform(input, {
+                parserOpts,
                 presets: [[_babili, babiliOpts]],
+                plugins,
```

Comment:
plugins are not required. This is not a `babel-webpack-plugin` this is `babili-webpack-plugin`. Babili has issues currently when used with plugins that change Scope. So, let's not have plugins in here.